### PR TITLE
install-deps.sh: add support for Ubuntu Disco Dingo

### PR DESCRIPTION
**Pull request by p-se**
Signed-off-by: Patrick Seidensal <pseidensal@suse.com>

## Checklist
- [ ] References tracker ticket
- [ ] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

---

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard backend`
- `jenkins test docs`
- `jenkins render docs`

</details>

### CODE REVIEW

File: install-deps.sh [hunk 1]
```diff
@@ -314,6 +314,10 @@ else
                 ensure_decent_gcc_on_ubuntu 9 bionic
                 [ ! $NO_BOOST_PKGS ] && install_boost_on_ubuntu bionic
                 ;;
+            *Disco*)
```
Comment (tchaikov) [inline install-deps.sh]: probably we don't need to install gcc-9 on ubuntu Disco. as it has GCC-8.3 already?

File: install-deps.sh [hunk 2]
```diff
@@ -314,6 +314,10 @@ else
...
+                ensure_decent_gcc_on_ubuntu 8 disco
```
Comment (tchaikov) [inline install-deps.sh]: @p-se i think we can simply drop this line once we have #30410 merged. as, on ubuntu Disco, we should have GCC-8. see https://packages.ubuntu.com/disco/gcc . so no need to install it from a PPA repo.

Comment (p-se) [reply install-deps.sh]: Yes, that has worked with the changes in #30410. I have remove that line. Does that work for other Ubuntu versions, too, or do they require a particular GCC version?

File: install-deps.sh (hunk 1 above)
Comment (p-se) [reply install-deps.sh]: Yes, Ubuntu Disco Dingo ships with GCC-8.3

File: install-deps.sh [hunk 3]
```diff
@@ -103,7 +103,7 @@ ENDOFKEY
             old=4.8;;
         xenial)
             old=5;;
-        bionic)
+        bionic|disco)
```
Comment (tchaikov) [inline install-deps.sh]: could drop this line also.

**Comment by tchaikov** [comment]
retest this please.

File: install-deps.sh (hunk 2 above)
Comment (tchaikov) [reply install-deps.sh]: i think i tried to answer your question in the commit message in #30410.

**Comment by tchaikov** [comment]
https://shaman.ceph.com/builds/ceph/wip-kefu-testing-2019-09-18-2145/68ad6a29364ad7825260de209c63dd669b26d244/

**Pull request merged**